spec/models/client_spec.rb

```diff
@@ -1,33 +1,39 @@
 require 'rails_helper'
 
 RSpec.describe Client, type: :model do
-  let!(:user) { create :user }
-  let!(:client) { create :client, :user => user }
-  let!(:message) { create :message, :user => user, :client => client}
+  describe 'relationships' do
+    let!(:user) { create :user }
+    let!(:client) { create :client, :user => user }
+    let!(:message) { create :message, :user => user, :client => client}
 
-  describe 'relationship to user' do
-    it 'belongs to a user' do
-      expect(client.user).to eq(user)
+    describe 'relationship to user' do
+      it 'belongs to a user' do
+        expect(client.user).to eq(user)
```

Comment:
Unrelated to this change, but I think testing that a user has certain relationships is veering pretty squarely into  "testing the framework" territory that isn't particularly valuable. For example, this will only break if you remove the `belongs_to` line in the model, or if Rails breaks that API terribly. Either way, not particularly useful at preventing bugs down the line.

I'm sure this is a controversial opinion though 😅